packages/app/src/client/components/withSession/index.tsx

```diff
@@ -313,10 +313,6 @@ const CreateSession = ({
       try {
         resp = await loginRequestWithRetry(at);
       } catch (err: any) {
-        if (!axios.isAxiosError(err)) {
-          // re-throw programming errors
-          throw err;
-        }
```

Comment:
This error was not handled anywhere. If this error is re-thrown like that, it will just result in a frozen UI with no feedback to the user.